### ISSUE TITLE
Mock failed

### ISSUE THREAD

**Issue by cpunion** [archan937/motion-bundler#7]
I want to mock `digest/md5` because `net/pop` depends it. My `mocks` directory tree is:

```
mocks/
    digest/
        md5.rb    # mock digest/md5
    net/
        protocol.rb    # mock net/protocol, just test
    socket.rb    # mock socket
```

The dependency is:

``` ruby
    "/Users/cpunion/.rvm/rubies/ruby-2.0.0-p0/lib/ruby/2.0.0/net/pop.rb" => [
      "/Users/cpunion/.rvm/rubies/ruby-2.0.0-p0/lib/ruby/2.0.0/net/protocol.rb",    # WRONG
      "/Users/cpunion/.rvm/rubies/ruby-2.0.0-p0/lib/ruby/2.0.0/x86_64-darwin12.4.0/digest/md5.bundle",    # WRONG
      "/Users/cpunion/.rvm/rubies/ruby-2.0.0-p0/lib/ruby/2.0.0/timeout.rb",
      "/Users/cpunion/.rvm/rubies/ruby-2.0.0-p0/lib/ruby/2.0.0/openssl.rb"
    ],
    "/Users/cpunion/.rvm/rubies/ruby-2.0.0-p0/lib/ruby/2.0.0/net/protocol.rb" => [
      "/Users/cpunion/test/motion/KanMail/mocks/socket.rb",  # CORRECT
      "/Users/cpunion/.rvm/rubies/ruby-2.0.0-p0/lib/ruby/2.0.0/timeout.rb"
    ]
```

I think it has some problems, or design?

**Comment by archan937** [comment]
Can you create a Github repository which I can checkout for debugging?

**Comment by cpunion** [comment]
Thank you for you quick reply.

Created at https://github.com/cpunion/motion-bundler-mock-debug

**Comment by archan937** [comment]
Thanks for the issue. I have fixed this in MotionBundler v0.1.6.

See also: https://github.com/archan937/motion-bundler-mock-debug/commit/eee07c1f25d6cd66f4368597a0ce8eeea62d7352

**Comment by cpunion** [comment]
Great! Thank you!
